selfdrive/ui/qt/maps/map_settings.cc

```diff
@@ -152,6 +152,13 @@ MapPanel::MapPanel(QWidget* parent) : QWidget(parent) {
       });
     }
   }
+
+  translateUi();
+}
+
+void MapPanel::translateUi() {
```

Comment:
I think Qt Designer names this function `retranslateUi`. Not a huge deal, but might want to fit in with that convention

Reply:
ok,I'll change it to retranslateUi later